resolwe_bio/tests/processes/test_expression.py

```diff
@@ -368,29 +368,23 @@ def test_feature_counts(self):
                 "annotation": annotation_gtf.id,
             },
         )
-        self.assertFile(
-            expression,
-            "rc",
-            outputs / "feature_counts_out_rc.tab.gz",
-            compression="gzip",
-        )
-        self.assertFile(
-            expression,
-            "cpm",
-            outputs / "feature_counts_out_cpm.tab.gz",
-            compression="gzip",
-        )
-        self.assertFile(
-            expression,
-            "exp",
-            outputs / "feature_counts_out_tpm.tab.gz",
-            compression="gzip",
-        )
-        self.assertFile(
-            expression,
-            "exp_set",
-            outputs / "feature_counts_out_exp_set.txt.gz",
-            compression="gzip",
+        self.assertDataFrameAlmostEqual(
```

Comment:
Why do we need assertDataFrameAlmostEqual here? FeatureCounts is a deterministic algorithm.